src/appleseed.python/CMakeLists.txt

```diff
@@ -44,7 +44,25 @@ endif ()
 # Source files.
 #--------------------------------------------------------------------------------------------------
 
-set (appleseed_module_sources
+set (studio_sources
+    studio/__init__.py
```

Comment:
oh, I forgot to add studio/Qt.py here in my commits. Could you please add it? or I can do it later